server.js

```diff
@@ -4625,6 +4644,16 @@ function objectWebServer() {
                                     thisObject.tcs = utilities.generateChecksums(objects, fileList);
                                     utilities.writeObjectToFile(objects, thisObjectId, objectsPath, globalVariables.saveToDisk);
                                     setAnchors();
+                                    
+                                    // Removes old heartbeat if it used to be an anchor
+                                    var oldObjectId = utilities.getAnchorIdFromObjectFile(req.params.id, objectsPath);
+                                    if (oldObjectId && oldObjectId != thisObjectId) {
+                                      console.log('removed old heartbeat for', oldObjectId);
+                                      clearInterval(activeHeartbeats[oldObjectId]);
+                                      delete activeHeartbeats[oldObjectId];
+                                      delete objects[oldObjectId];
+                                    }
```

Comment:
This won't be necessary if we fix the root problem around anchors' object.json and their ID changing, but good for now.